github/auth.go

```diff
@@ -223,6 +224,7 @@ func NewClient(ctx context.Context, optFns ...ClientOption) (gitprovider.Client,
 	// the default.
 	var gh *github.Client
 	var domain string
+
```

Comment:
I started adding all these spaces because of the whitespace linter wsl, but then was just like "nah, I don't care" and left half-way through :joy:
I think I'll live things as-is now, and someone can later make whitespace consistent across the project.
Low-hanging fruit anyways.